src/avram/query_builder.cr

```diff
@@ -158,6 +159,11 @@ class Avram::QueryBuilder
     self
   end
 
+  def for_update : self
+    @for_update = true
```

Comment:
Postgres supports more than just `FOR UPDATE` 

> FOR { UPDATE | NO KEY UPDATE | SHARE | KEY SHARE }

So this might need some sort of args, or maybe just different methods like

```crystal
def for_no_key_update
end
def for_share
end
def for_key_share
end

# VS

def for_update(for_method)
end
for_update(:no_key_update)
for_update(:share)
```